src/forms/EditDocument/ExistingDocsMessage.js

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from 'react';
+import {sortFilesBySimilarity} from '../../common/fileUtils';
+import {Message, List, Icon, Button} from 'semantic-ui-react';
+
+const ExistingDocsMessage = ({
+  existingDocs,
+  fileNameInput,
+  errors,
+  setShowDialog,
+}) => {
+  const [showSimilar, setShowSimilar] = useState(false);
+  const similarStudyDocs = sortFilesBySimilarity(
+    {name: fileNameInput},
+    existingDocs,
+    0.33,
+  );
+  const SimilarTitleContent = () => (
+    <>
+      We found
+      <strong>
+        {similarStudyDocs.matches.length ? similarStudyDocs.matches.length : ''}{' '}
+        similar document titles
+      </strong>{' '}
+      in this study.
+      <a
+        href=" "
+        onClick={e => {
+          e.preventDefault();
+          setShowSimilar(!showSimilar);
+        }}
+      >
+        {' '}
+        {!showSimilar ? 'Show' : 'Hide'} Similar
+      </a>
+      {showSimilar && (
+        <List>
+          {similarStudyDocs.matches.map(doc => (
```

Comment:
These don't render for me, even though the message does.

Reply:
they don't render when you click `Show Similar`?